.github/workflows/CI.yml

```
@@ -14,7 +14,10 @@ jobs:
       - uses: actions/checkout@v2
       - uses: julia-actions/setup-julia@v1
         with:
-          version: 1
+          version:
+            - '1.8'
+            - '1.9'
+            - 'nightly'
```

```suggestion
            - 1.8
            - 1.9
            - nightly
```

I think without quotes? ci didn't run before... let's try this way